### DISCUSSION TITLE
[WARNING] No animation called "singUP"/"singDOWN"/"singLEFT"/"singRIGHT"

### DISCUSSION THREAD

**Discussion by kinniechaos** [KadeArchive/Kade-Engine#1041] [Q&A]
Not sure if this place is the best place to put this, but I'm tryin to mod over another character's sprites from another mod already made with kade engine, and instead of using the mod itself I downloaded the source code since I knew I was gonna have to fix offsets. I used the same FLA files that had the original character's animations, I just replaced his animations with the one I'm reskinning him with. Now whenever I compile the game, everything runs fine, but instead of any of the other animations playing it only uses his idle. I took a look, and the only animations that work are his idle and an alt animation. Everything in Character.hx is fine to my knowledge, and all of the animations have the right names, both in the XML and Character.hx. Any suggestions? I'm assuming I somehow accidentally broke the code like the dumbass I am, but just in case it was an issue with the engine itself I wanted to try here.

**Comment by MoezTrigui** [comment]
it isnt did you see the capitals

**Comment by kinniechaos** [comment]
> it isnt did you see the capitals
What? Sorry, I don't really understand what you mean by that
